python/agents/application/trade.py

```
@@ -0,0 +1,76 @@
+            print()
+            print(f"3. FOUND {len(markets)} MARKETS")
+
+            print()
+            filtered_markets = self.agent.filter_markets(markets)
+            print(f"4. FILTERED {len(filtered_markets)} MARKETS")
+
+            market = filtered_markets[0]
+            best_trade = self.agent.source_best_trade(market)
+            print(f"5. CALCULATED TRADE {best_trade}")
+
+            amount = self.agent.format_trade_prompt_for_execution(best_trade)
```

Ruff (0.11.9)

Local variable `amount` is assigned to but never used

Remove assignment to unused variable `amount`

(F841)